pipelines/ceus/tasks/fcz_correlation.py

```diff
@@ -50,6 +53,9 @@ def _task(self):
         correlation_metrics = ['Temperature', 'Solar Zenith Angle', 'GHI', 'DHI', 'DNI', 'Wind Speed', 'Wind Direction', 'Relative Humidity']    
         correlation_matrix = pd.DataFrame(0, index=self.projection_locations, columns=self.fcz_names)
 
+        self.fcz_names = [x for x in self.fcz_names if x not in self.excluded_locations['base']]
+        self.projection_locations = [x for x in self.projection_locations if x not in self.excluded_locations['target']]
```

Comment:
This works, but it would still be simpler to use the set difference. What was your reasoning to choose one over the other?